Sources/Contentful/Decodable.swift

```diff
@@ -17,11 +17,11 @@ public extension Decoder {
 
     /// The `TimeZone` the `Decoder` is using to offset dates by.
     /// Set through `ClientConfiguration`.
-    public var timeZone: TimeZone? {
+    var timeZone: TimeZone? {
```

Comment:
I am not sure if changing the access control from public to internal is needed by the array nesting issue.

Do you mind helping me understand why we need this change?

Reply:
Decodable is a public interface, Swift compiler is complaining that `public` modifier is unnecessary.